exporter/azureblobexporter/factory.go

```diff
@@ -42,49 +43,57 @@ func createDefaultConfig() component.Config {
 			Traces:  "traces",
 		},
 		BlobNameFormat: &BlobNameFormat{
-			MetricsFormat:  "2006/01/02/metrics_15_04_05_{{.SerialNum}}.{{.FileExtension}}",
-			LogsFormat:     "2006/01/02/logs_15_04_05_{{.SerialNum}}.{{.FileExtension}}",
-			TracesFormat:   "2006/01/02/traces_15_04_05_{{.SerialNum}}.{{.FileExtension}}",
+			MetricsFormat:  "2006/01/02/metrics_15_04_05.json",
```

Comment:
Should default have millisecond precision? For large workloads.

Reply:
There's also serial number, maybe minutes is enough as default?

Reply:
I also plan to support append blob feature, but will do it in another PR for future.